docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,339 @@
+  * `Navigation`- Interface responsible for RPC events and information related to navigation. Includes `StartStream` and `GetWayPoints`.
+  * `VehicleInfo`- Interface responsible for RPC events and information related to vehicle data. Includes `GetVehicleData` and `SubscribeVehicleData`.
+  * `RC` - Interface responsible for RPC events and information related to the [Remote Control Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0065-remote-control.md).
+  * `AppService` - Interface responsible for RPC events and information related to the [App Services Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0167-app-services.md).
+
+### Response
+
+SDL provides a JSON Response
+
+| Key | Value Info   |
+| :------------- | :------------- |
+| "id"      | The value from the corresponding request      |
+| "result" | Value of id multiplied by 10. HMI can treat this as a successful registration|
+
+Example Response:
+``` json
+{
+  "id": 100,
+  "jsonrpc": "2.0",
+  "result": 1000
+}
+```
+
+## Component Readiness Requests 
+
+Once the components are registered, the HMI must notify SDL Core that it is ready to begin further communication using the [BasicCommunication.OnReady](../basiccommunication/onready) notification.
+
+Upon receipt of the OnReady notification, SDL Core will begin checking the availability of the different HMI components via a chain of requests:
+
+  * `UI.IsReady` - The display availability
+  * `VR.IsReady` - The voice recognition module availability
+  * `TTS.IsReady` - The Text-To-Speech module availability
+  * `Navigation.IsReady` - Navigation engine availability
+  * `VehicleInfo.IsReady` - Indicates whether vehicle information can be collected and provided
+  * `RC.IsReady` - Indicates whether vehicle RC modules are present and ready to communicate with SDL Core
+
+!!! note
+
+In the case of a WebSocket connection, RPCs to each of the components are sent within a separate WebSocket connection.
+
+!!!
+
+|||
+IsReady Sequence
+![IsReady Sequence](./assets/IsReadySequence.png)
+|||
+
+!!! info
+
+If the response to any of the component `IsReady` requests contains `{"available": false}`, SDL Core will no longer communicate with that component.
+
+!!!
+
+ 
+## Registering for Notifications
+The HMI must also register for notifications individually using the following RPC format.
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": -1,
+  "method": "MB.subscribeTo",
+  "params": {
+    "propertyName": notificationName
+  }
+}
+```
+
+"propertyName" is the name of the notification the HMI will receive from Core. Some examples include:
+
+  * Buttons.OnButtonSubscription
+  * BasicCommunication.OnAppRegistered
+  * BasicCommunication.OnAppUnregistered
+  * Navigation.OnVideoDataStreaming
+  * SDL.OnStatusUpdate
+
+Core's message broker will not route notifications to the HMI unless the notifications are subscribed to.
```

Comment:
In [line 56](https://github.com/smartdevicelink/sdl_core_guides/pull/59/files#diff-aad4ca0433fcaa1d53a08891f7587b7bR56) you use `MessageBroker`.  These should be consistent.